configs/talos_config.json

```diff
@@ -0,0 +1,10 @@
+{
+    "applabel": "['talos-jobs']",
```

Comment:
Could you please change this to "worker3"? I've found out that the labels don't have to match the keys of the exchanges.